cppfd/CMakeLists.txt

```diff
@@ -3,11 +3,20 @@ cmake_minimum_required(VERSION 3.15)
 # Set the project name
 project(cfd-mini-app)
 
+# Setting configure file
+configure_file(cfd_config.h.in cfd_config.h)
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Probably instead of generating config file and keeping it included, you could just do `add_definitions(-DOUTPUT_VTK_FILES)` https://cmake.org/cmake/help/v3.15/command/add_definitions.html

Reply:
Haven't got time to check, Philippe was faster :)